backend/SS_Position/core.py

```diff
@@ -24,8 +21,18 @@ def processMovement(player, key):
 
     resultPosition = str(x) + " " + str(y)
 
-    if mapWorker.checkPositionMovable(x,y):
+    closed = checkDoorClosing(baseX,baseY)
+    if closed != "":
+        redisWorker.setPlayerPosition(player,x,y)
+        return  "CLOSE " + str(x) + " " + str(y) + " " + closed
+    elif mapWorker.checkPositionMovable(x,y):
         redisWorker.setPlayerPosition(player,x,y)
-        return "MOVE " + str(x) + " " + str(y) + " " + door_closed
+        return "MOVE " + str(x) + " " + str(y)
     else:
         return "UPD " + resultPosition + " " + mapWorker.processOpenable(x,y)
+
+def checkDoorClosing(x, y):
```

Comment:
So, if this function starts with "check...", then, shouldn't it return boolean value?